rasa_core/training/dsl.py

```diff
@@ -305,11 +305,17 @@ def add_user_messages(self, messages, line_num):
         self.current_step_builder.add_user_messages(parsed_messages)
 
     def add_event(self, event_name, parameters):
-        if "name" not in parameters:
+
+        if "name" not in parameters and event_name != SlotSet.type_name:
```

Comment:
this is a bit hacky, but I don't see any other way right now :+1:

please add a comment ala "need to do this ... slot might be named name ...."